molgenis-security/src/main/java/org/molgenis/security/MolgenisSecurityWebAppDatabasePopulatorServiceImpl.java

```diff
@@ -114,7 +106,8 @@ public void populateDatabase(DataService dataService, String homeControllerId)
 		// allow all users to read the app languages
 		GroupAuthority usersGroupLanguagesAuthority = groupAuthorityFactory.create();
 		usersGroupLanguagesAuthority.setMolgenisGroup(allUsersGroup);
-		usersGroupLanguagesAuthority.setRole(SecurityUtils.AUTHORITY_ENTITY_READ_PREFIX + LANGUAGE.toUpperCase());
+		// FIXME do not hardcode default language code
+		usersGroupLanguagesAuthority.setRole(SecurityUtils.AUTHORITY_ENTITY_READ_PREFIX + "EN");
```

Comment:
Not here, this is the name of the entity (will merge and then put up new PR to change)